### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
For the third milestone of my portfolio website, I did:

contact form to my portfolio and the portfolio of my coding partner.
I am sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
and I  used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
and as well as my  documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -230,5 +230,79 @@ <h2>About Me</h2>
...
+                </h6>
+                <form id="form" action="https://formspree.io/f/moqrjalz" method="post">
+                  <input
+                    name="fullname"
+                    id="fullname"
+                    class="stored"
+                    type="text"
+                    placeholder="Full name"
+                    minlength="5"
+                    maxlength="50"
```
Comment (awais-amjed) [inline index.html]: Kindly change the `maxlength` to 30 following the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) 🤗

File: index.html [hunk 2]
```diff
@@ -230,5 +230,79 @@ <h2>About Me</h2>
...
+                    so if you'd like to chat please get in touch.
+                </h6>
+                <form id="form" action="https://formspree.io/f/moqrjalz" method="post">
+                  <input
+                    name="fullname"
+                    id="fullname"
+                    class="stored"
+                    type="text"
+                    placeholder="Full name"
+                    minlength="5"
```
Comment (awais-amjed) [inline index.html]: [OPTIONAL] It's better to remove `minlength` since some people might have a shorter name e.g. Bob

File: index.html [hunk 3]
```diff
@@ -230,5 +230,79 @@ <h2>About Me</h2>
...
+                    type="email"
+                    placeholder="Email address"
+                    aria-label=" Email Address"
+                    required
+                  />
+                  <textarea
+                    name="message"
+                    id="message"
+                    class="stored"
+                    minlength="5"
```
Comment (awais-amjed) [inline index.html]: [OPTIONAL] It's better to remove `minlength` since some people might just wanna say Hi 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @ZuhRa-HashiMi , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html (hunk 3 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — APPROVED** [review]
Hi @ZuhRa-HashiMi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**